UserTools/LikelihoodFitterCheck/LikelihoodFitterCheck.cpp

```diff
@@ -73,6 +99,15 @@ bool LikelihoodFitterCheck::Execute(){
   	Log("LikelihoodFitterCheck  Tool: Error retrieving RecoDigits,no digit from the RecoEvent!",v_error,verbosity); 
   	return false;
   }
+
+  if (fUsePDFFile) {
+      bool pdftest = this->GetPDF(pdf);
+      if (!pdftest) {
+          Log("LikelihoodFitterCheck  Tool: Error retrieving pdffile; running without!", v_error, verbosity);
+          fUsePDFFile = 0;
+          return false;
```

Comment:
should this return false? The printout suggests that it will continue without, but it doesn't. At least, not until the next Execute call.